services/src/messaging/src/main/java/org/openkilda/messaging/info/switches/PortConfigurationResponse.java

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.switches;
+
+import org.openkilda.messaging.info.InfoData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Getter
+@Setter
+public class PortConfigurationResponse extends InfoData {
+
+    private static final long serialVersionUID = 7332950619136252164L;
+
+    @JsonProperty("switch_id")
+    private String switchId;
+
+    @JsonProperty("port_no")
+    private int portNo;
+
+    @JsonProperty("successs")
+    private boolean successs;
```

Comment:
It is redundant here. You can have this field in DTO returned from NB API call, but here it is redundant. (In case of error you are returning other object).

Reply:
+1

Reply:
How a client can get the failure reason in the case of "successs=false"?

Reply:
Yes, Updated code not this success message is to return whether a message is pushed on the switch or not.

Reply:
Don't see any update here. 

PS I can't understand this `Updated code not this success message is to return whether a message is pushed on the switch or not.` sentence.
PPS There is not so many way for you to detect what goes wrong... but you can at least say when you have failed to write command into switch.